articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+
+## Collateral Risk
+
+In the beginning, the only collateral enabled to borrow XAI will be USDC and ETH. Thus, eliminating other risks that come with long tail assets (e.g., low asset liquidity, high price volatility, depeg of the collateral, etc.).
+
+Both ETH and USDC are the highest-tier assets with more than enough liquidity to offset potential liquidations. Even when XAI will be enabled as a bridge asset for more silos, the collateral risk is limited only to those isolated markets.
+
+We currently don’t see any risks related to collateral, however, anyone can propose to add new credit lines. [Credit lines](https://silopedia.silo.finance/welcome/cross-silos-stablecoin) describe the process of allowing silos to use XAI as a bridge asset. While new credit lines need to be approved by governance first, this process can change the composition of XAI’s backing. Potential risks occur in cases where XAI supports illiquid and highly volatile assets. Hence, we recommend that each credit line and silo addition be carefully considered. Silo should also think about installing or incentivizing more detailed risk assessments for each collateral. A bad debt dashboard (as provided by RiskDAO for instance), is another option to better inform users about the health of individual silos.
+
+
+## XAI as Collateral Risk
+
+Silo also aims to enable XAI as collateral within the protocol. As highlighted above, for XAI to become a low-risk collateral, there needs to be enough liquidity in the open market. This will help to facilitate potential liquidations and arbitrage, which is needed to support XAi’s stability.
+
+The team is working on two [initiatives](https://gov.silo.finance/t/building-on-chain-liquidity-for-xai/309) that will ensure the initial provisions of adequate liquidity. First, seeding the initial Curve pool with $3M (50% USDC / 50% XAI) from their development fund. And secondly, Silo will use 130k of their own vlCVX to vote for incentivization of their pool. Hence, we believe there will be enough liquidity for XAI’s early stage.
+
+
+# Discussion and Conclusion
+
+
+
+1. Is it possible for a single entity to rug its users?
+
+No, SiloDAO does not have access to user funds. It can, however, mint XAI to silos or redeem XAI. The DAO also controls the interest rates, LTV, and other parameters of XAI. Thus, having a strong influence on the price stability of XAI.
```

Comment:
reiterate high concentration of gov power to founding team

Reply:
done